lib/tasks/recalculate_irproduct_values.rake

```diff
@@ -0,0 +1,18 @@
+desc "Forces to recalculate irproduct values"
+task recalculate_irproduct_values: :environment do
+  LettingsLog.exportable.where(rent_type: [3, 4, 5]).where(irproduct: nil).each do |log| # irproduct was never set
+    Rails.logger.info("Could not update irroduct for LettingsLog #{log.id}") unless log.update(values_updated_at: Time.zone.now)
+  end
+  LettingsLog.exportable.where(rent_type: 3).where.not(irproduct: 1).each do |log| # irproduct was set wrong
+    Rails.logger.info("Could not update irproduct for LettingsLog #{log.id}") unless log.update(values_updated_at: Time.zone.now)
+  end
+  LettingsLog.exportable.where(rent_type: 4).where.not(irproduct: 2).each do |log| # irproduct was set wrong
+    Rails.logger.info("Could not update irproduct for LettingsLog #{log.id}") unless log.update(values_updated_at: Time.zone.now)
+  end
+  LettingsLog.exportable.where(rent_type: 5).where.not(irproduct: 3).each do |log| # irproduct was set wrong
+    Rails.logger.info("Could not update irproduct for LettingsLog #{log.id}") unless log.update(values_updated_at: Time.zone.now)
+  end
```

Comment:
There are none of these on prod atm, but if rent_type is changed, these could happen